me-lazyload.js

```diff
@@ -99,6 +100,19 @@ angular.module('me-lazyload', [])
 
             iElement.bind('load', onLoad);
 
+            var $scrollContainer = $win
+            $scrollContainer.on('scroll', checkImage);
+
+
+            $scope.$watch('scrollContainer', function(newVal){
+                if(newVal){
+                    $scrollContainer.off('scroll', checkImage);
+                    $scrollContainer = angular.element(document).find($scope.scrollContainer);
+                    $scrollContainer.on('scroll', checkImage);
```

Comment:
If the scrollContainer is `#container`, and it have multi images children.
Then it will bind the same checkImage function everytime a new lazy-src directive created.

Because when you invoke `off` event, the `scrollContainer` is still window, then you set scrollContainer to `#container`, and then bind `checkImage`. Everytime lazySrc created, the same thing is done.